.github/workflows/tests.yml

```diff
@@ -241,6 +241,7 @@ jobs:
             --added-formulae="$ADDED_FORMULAE" \
             --deleted-formulae="$DELETED_FORMULAE"
         env:
+          HOMEBREW_LIVECHECK_AUTOBUMP: 1
```

Comment:
Seems like something that should be set in `test-bot` instead of here.

Reply:
I wasn't sure which would be preferable but that makes sense to me. One moment and I'll open a PR.

Reply:
PR is now up at https://github.com/Homebrew/homebrew-test-bot/pull/1353 and I've set this one as a draft. I'll close this PR if/when we go forward with the test-bot approach.